EarTrumpet/EarTrumpet.csproj

```diff
@@ -174,4 +175,4 @@
   <Target Name="AfterBuild">
   </Target>
   -->
-</Project>
\ No newline at end of file
+</Project>

```

Comment:
Whitespace change, should be reverted.

This is a generated file too which is going to be regularly modified by Visual Studio, so chances are the next time a change is made to this file by a built-in VS function, it'll likely revert this back to having no newline at the end of the file, so we should leave it in the format the tool expects it in otherwise it'll keep changing and cause noise in diffs.

It's nasty, I know, but a necessary evil with tool-generated and tool-modifiable files...